guides/source/action_controller_overview.md

```diff
@@ -131,11 +131,11 @@ To send a hash you include the key name inside the brackets:
 
 When this form is submitted, the value of `params[:client]` will be `{ "name" => "Acme", "phone" => "12345", "address" => { "postcode" => "12345", "city" => "Carrot City" } }`. Note the nested hash in `params[:client][:address]`.
 
-Note that the `params` hash is actually an instance of `ActiveSupport::HashWithIndifferentAccess`, which acts like a hash but lets you use symbols and strings interchangeably as keys.
+The `params` hash is actually an instance of `ActiveSupport::HashWithIndifferentAccess`, which acts like a Hash but lets you use symbols and strings interchangeably as keys.
```

Comment:
This is not true true anymore. `params` is an instance of `ActionController::Parameters`. Maybe we should not talk about which class it inherit but only that symbols and strings are interchangeable